config.coffee

```diff
@@ -53,7 +53,7 @@ module.exports =
   API_REQUEST_TIMEOUT: 5000
   FORCE_URL: 'https://www.artsy.net'
   PREDICTION_URL: 'https://live.artsy.net'
-  METAPHYSICS_ENDPOINT: 'https://metaphysics-production.artsy.net'
+  METAPHYSICS_ENDPOINT: 'http://localhost:5001'
```

Comment:
Lol just saw this, silly GitHub review feature. I was actually pointing to my local metaphysics in the event that I needed to make some updates there as well. I find it's easier to get the pieces working together and then open the PR's in both, rather than going back and forth.